package.json

```diff
@@ -27,7 +27,9 @@
   "license": "SEE LICENSE IN https://tc39.es/ecma262/#sec-copyright-and-software-license",
   "homepage": "https://tc39.es/ecma262/",
   "dependencies": {
-    "ecmarkup": "9.6.3"
+    "ecmarkup": "9.6.3",
+    "cpy-cli": "^3.1.1",
```

Comment:
Installing this package adds 49 transitive dependencies, which seems excessive for such a trivial utility. While I'm not opposed to using a dependency for this, I would prefer something with fewer transitive dependencies, ideally zero.

Reply:
I found and applied another package ([cpr](https://www.npmjs.com/package/cpr)) with almost no dependencies:

- graceful-fs (has no dependencies)
- minimist (has no dependencies)
- mkdirp (has no dependencies)
- rimraf (already a dependency of ecma262 itself)